buildsuites.xml

```diff
@@ -288,6 +292,38 @@
         />
     </target>
 
+    <!-- Regression tests in broad servers -->
+    <target name="short-nightly-tests-broad" depends="package, clone-test-repos">
+        <union id="short.tests.beta.ami">
+            <resources refid="ShortComparativeMarkerSelectionTests" />
+            <resources refid="ShortCoxRegressionTests" />
+            <resources refid="ShortExpressionFileCreatorTests" />
+            <resources refid="ShortHeatMapImageTests" />
+            <resources refid="ShortHeatMapViewerTests" />
+            <resources refid="ShortHierarchicalClusteringTests" />
+            <resources refid="ShortTopHatTests" />
+            <!-- <resources refid="Shorttxt2odfTests" /> --> <!-- Removing this from Broad, we are not putting new modules on old servers-->
+            <resources refid="nightlyMergeHTSeqCountsTests" />
+            <resources refid="nightlyPreprocessDatasetTests" />
+            <resources refid="nightlyPreprocessReadCountsTests" />
+        </union>
+        <!-- The GpUnit code parses filesets using the value returned by a call to File.pathSeparator() -->
+    	<!-- as the delimiter. This returns ";" on Windows and ":" on Linux. However, ant creates filesets -->
+    	<!-- with ";" as the separator on both platforms (!). The pathconvert task transforms the fileset -->
+    	<!-- to one with a delimiter that properly matches what the code expects. -->
+        <pathconvert property="short.nightly.tests" refid="short.tests" />

```

Comment:
is this the correct refid?

Reply:
I think this is an overlook on my end. line 314 should be:
`<pathconvert property="short.nightly.tests.broad" refid="short.tests.broad" />`
right?

Reply:
correct